vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -24,8 +24,8 @@
 #include "NativeModulesProvider.h"
 #include "Unicode.h"
 
-#include <ReactWindowsCore/DevServerHelper.h>
-#include <ReactWindowsCore/ViewManager.h>
+#include <Shared/DevServerHelper.h>
```

Comment:
>Shared [](start = 10, length = 6)

I wonder why do we need the Shared project. Why not to have all these files as a part of Microsoft.ReactNative project?

Reply:
Because the Desktop variant doesn’t build on top of MSRN.

This change doesn’t affect the project structure but only consolidates directories.